templates/telemetry/account-ecosystem/account-ecosystem.4.schema.json

```diff
@@ -3,6 +3,14 @@
   "type": "object",
   "title": "account-ecosystem",
   "mozPipelineMetadata": {
+    "bq_access": {
+      "workgroups": [
+        {
+          "members": [],
```

Comment:
This empty list seems strange. Should we just have `"workgroups": []` in this case where we're allowing only administrator-level access?